app/views/admin/split_contexts/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-container sc-m-t--l"%>
+  <%= simple_form_for @split, url: admin_split_context_path, method: :put do |f| %>
+    <%= f.error :base %>
+    <%= f.input :hypothesis, hint: "What do we expect to change with the treatment condition, and why?" %>
+    <%= f.input :context, hint: "When do we assign users to this split? Is the split population constrained to a particular type of user or action? Assignments should happen as \"close\" to the varied experience as possible, so that only relevant users are included in the analysis." %>
```

Comment:
I wonder if we want to double up on our usage of `context` in TT...it's already an attribute on `assignments`.  Perhaps `population` or even just a more descriptive `assignment_context`?